--- a/src/getopt.h.orig	2026-04-07 08:42:50.279647438 -0300
+++ b/src/getopt.h	2026-04-07 08:44:02.731647411 -0300
@@ -105,7 +105,7 @@
    errors, only prototype getopt for the GNU C library.  */
 extern int getopt (int argc, char *const *argv, const char *shortopts);
 #else /* not __GNU_LIBRARY__ */
-extern int getopt ();
+
 #endif /* __GNU_LIBRARY__ */
 extern int getopt_long (int argc, char *const *argv, const char *shortopts,
                         const struct option *longopts, int *longind);
